syncd/Syncd.cpp

```diff
@@ -203,9 +203,9 @@ void Syncd::performStartupLogic()
 {
     SWSS_LOG_ENTER();
 
-    // ignore warm logic here if syncd starts in Mellanox fastfast boot mode
+    // ignore warm logic here if syncd starts in fastboot or in Mellanox fastfast boot mode
 
-    if (m_isWarmStart && (m_commandLineOptions->m_startType != SAI_START_TYPE_FASTFAST_BOOT))
+    if (m_isAdvancedStart && m_commandLineOptions->m_startType == SAI_START_TYPE_WARM_BOOT)
```

Comment:
changing fastboot to warm boot may have some issues here, or is this the way it should be ?

Reply:
rest looks good, please resolve issues, or because of the type of this change it cant befixed? is this breaking change and not backward compatible?